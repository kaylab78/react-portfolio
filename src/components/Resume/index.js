import React from 'react';

function Resume() {
    return (
        <section>
            <h1 id="resume">Resume</h1>
            <a href="https://drive.google.com/file/d/1gpUjajV0Jw6UU7kf0Xb-MQ--5O3nEnri/view?usp=sharing" target="_blank" rel="noreferrer" className="project-link">Download my resume.</a>
            <h2>Skills</h2>
            <ul>
                <li>HTML5</li>
                <li>CSS3</li>
                <li>JavaScript</li>
                <li>APIs</li>
                <li>Node.js</li>
                <li>Express.js</li>
                <li>SQL</li>
                <li>MySQL</li>
                <li>Bootstrap</li>
                <li>Wordpress</li>
                <li>Oxygen Builder</li>
                <li>Woocommerce</li>
                <li>Stripe</li>
                <li>TownNews CMS</li>
                <li>GitHub</li>
                <li>Git</li>
                <li>Heroku</li>
                <li>VS Code</li>
                <li>Insomnia</li>
            </ul>
        </section>
    )
}

export default Resume;