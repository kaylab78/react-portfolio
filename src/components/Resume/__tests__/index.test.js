import React from 'react';
import { render, cleanup, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';

afterEach(cleanup);

describe('Resume component', () => {
    it('renders', () => {
        render(<Resume />);
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<Resume />);
        
        expect(asFragment()).toMatchSnapshot();
    });
});

// Test template from App.test.js that the Create React App CLI came with.
test('renders download my resume link', () => {
    render(<Resume />);
    const linkElement = screen.getByText(/Download my resume./i);
    expect(linkElement).toBeInTheDocument();
});