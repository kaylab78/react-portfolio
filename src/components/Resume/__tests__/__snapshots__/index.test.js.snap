// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="resume"
    >
      Resume
    </h1>
    <a
      href="https://drive.google.com/file/d/111ou3cnN_gI37WTGRt6mHbKfyMoHSBm6/view"
      rel="noreferrer"
      target="_blank"
    >
      Download my resume.
    </a>
    <h2>
      Skills
    </h2>
    <ul>
      <li>
        HTML5
      </li>
      <li>
        CSS3
      </li>
      <li>
        JavaScript
      </li>
      <li>
        APIs
      </li>
      <li>
        Node.js
      </li>
      <li>
        Express.js
      </li>
      <li>
        SQL
      </li>
      <li>
        MySQL
      </li>
      <li>
        Bootstrap
      </li>
      <li>
        Wordpress
      </li>
      <li>
        BLOX CMS
      </li>
      <li>
        GitHub
      </li>
      <li>
        Git
      </li>
      <li>
        Heroku
      </li>
      <li>
        VS Code
      </li>
      <li>
        Insomnia
      </li>
    </ul>
  </section>
</DocumentFragment>
`;
